hail/python/hail/docs/hailpedia/matrix_table.rst

```diff
@@ -340,7 +343,7 @@ Next we group the columns by `case_status` and aggregate:
         }
     }
     >>> print(mt_grouped.col.dtype)
-    struct{status: str}
+    struct{case_status: str}
```

Comment:
nice, this is exactly what I love doctests for!

Reply:
👍